test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -184,6 +262,53 @@ public void windowClosing(WindowEvent e) {
         windowList.add(frame);
     }
 
+    private static void captureScreen(Rectangle bounds) {
+        if (robot == null) {
+            try {
+                robot = new Robot();
+            } catch (AWTException e) {
+                String errorMsg = "Failed to create the " +
+                        "instance of Robot.";
+                JOptionPane.showMessageDialog(frame, errorMsg, "Failed",
+                        JOptionPane.ERROR_MESSAGE);
+                forceFail(errorMsg + e.getMessage());
+            }
+        }
+        MultiResolutionImage multiResolutionImage = robot
+                .createMultiResolutionScreenCapture(bounds);
+        List<Image> imageList =
+                multiResolutionImage.getResolutionVariants();
+        Image image = imageList.get(imageList.size() - 1);
```

Comment:
```suggestion
        MultiResolutionImage mri = robot.createMultiResolutionScreenCapture(bounds);
        List<Image> imageList = mri.getResolutionVariants();
        Image image = imageList.get(imageList.size() - 1);
```
`mri` is a common abbreviation of `MultiResolutionImage`, it's much shorter.

Alternatively, you may inline the two calls:
```suggestion
        List<Image> imageList = robot.createMultiResolutionScreenCapture(bounds)
                                     .getResolutionVariants();
        Image image = imageList.get(imageList.size() - 1);
```